tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -704,6 +706,20 @@ public void resume() {
     requestLayout();
   }
 
+  /**
+   * Allows the TileView to render tiles while panning.
+   *
+   * @param shouldRender True if it should render while panning.
+   */
+  public void setShouldRenderWhilePanning( boolean shouldRender ) {
+    mShouldRenderWhilePanning = shouldRender;
+    if( shouldRender ) {
```

Comment:
when a condition just determines a value (rather than an "action" - think: method), i think ternary is appropriate:

```
int buffer = shouldRender ? TileCanvasViewGroup.FAST_RENDER_BUFFER : TileCanvasViewGroup.DEFAULT_RENDER_BUFFER;
mTileCanvasViewGroup.setRenderBuffer( buffer );
```